openhtf/io/frontend/__main__.py

```diff
@@ -96,47 +96,41 @@ class StationStore(threading.Thread):
   """
   def __init__(self):
     super(StationStore, self).__init__()
-    self._death_event = threading.Event()
+    self._stop_event = threading.Event()
     self.hostname = socket.gethostname()
     self.stations = collections.defaultdict(Station)
```

Comment:
ah, yea i wasn't sure how much you cared about having multiple stations with the same id show up.  in that case, how about adding host and port to the `Station` class and moving `QueryStation` and `NotifyStation` there, rather than having them be `@staticmethod`s on `StationStore`?  You can still keep the `StationStore.stations` map the same, but it removes a little bit of clutter from this class.